webrtc.html

```diff
@@ -3885,19 +3901,48 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
                   </ul>
                 </li>
                 <li>
-                  <p>If <var>sender</var> was left unset by the previous step,
-                  create a new <code><a>RTCRtpTransceiver</a></code> with
-                  <code>sender.track</code> set to <var>track</var>, add it to
-                  <var>connection</var>'s set of transceivers.</p>
-                  <p>Let <var>sender</var> be the
-                  <code><a>RTCRtpSender</a></code> object associated with the
-                  newly created <code><a>RTCRtpTransceiver</a></code>.</p>
-                  <p>The initial value of an
-                  <code><a>RTCRtpTransceiver</a></code>'s <code><a data-for=
-                  "RTCRtpTransceiver">mid</a></code> attribute is null. Setting
-                  a new <code><a>RTCSessionDescription</a></code> may change it
-                  to a non-null value, as defined in <span data-jsep=
-                  "processing-a-local-desc processing-a-remote-desc">[[!JSEP]]</span>.</p>
+                  <p>If <var>sender</var> was set by the previous step, run the
+                  following steps to use that sender:</p>
```

Comment:
`sender` is *optionally set* in the previous paragraph, which I found a bit hard to reason about. Rather than referencing previous logic, would it be simpler to bake it in, i.e. initialize it explicitly in it its own step?

  * *Let* sender *be* *`null`*.

And then here: *If* sender *is not `null`, run the following steps to use that sender:*

Also (github wont let me comment on lines directly above this patch):
* since you're in here, if you could change *"matches the following"* to *"matches all the following"*, I think it would be clearer (I tripped up on the "any" in the same sentence).
* The matching criteria references "sender", which is not `sender`, which is logically ok, but a bit confusing.